configs/Ec2UserdataUtils.psm1

```diff
@@ -1434,7 +1434,7 @@ function Create-Hgrc {
 function Install-BuildBot {
   param (
     [string] $version = '21b5392348c3', # latest: 'default'
-    [string] $url = ('https://hg.mozilla.org/build/puppet/raw-file/{0}/modules/buildslave/files/runslave.py' -f $version),
+    [string] $url = ('https://raw.githubusercontent.com/mozilla/build-puppet/master/modules/buildslave/files/runslave.py' -f $version),
```

Comment:
You are dropping the `{0}` here. We should probably keep this the same as previously. I've commented with what I think the correct corresponding git commits, but somebody should double check me.

Reply:
Guh, sorry about that. Thanks for catching it.